packages/frontend/src/components/chart/configure/render/renderHover.ts

```diff
@@ -137,3 +164,42 @@ function renderTpsRow(value: number, source: 'L2' | 'ETH') {
   const circleHTML = `<div class="${circleClass}"></div>`
   return `<div>${circleHTML} ${sourceHTML} avg. TPS: ${formattedHTML}</div>`
 }
+
+function renderNameRow(name: string) {
+  return `<div class="mb-2 font-bold flex flex-wrap"><svg class="absolute mt-1 dark:fill-green-500 dark:stroke-white fill-green-600 stroke-green-200" width="11" height="10" viewBox="0 0 11 10">
+  <rect x="5.24268" y="0.0502174" width="7" height="7" rx="1.5" transform="rotate(45 5.24268 0.0502174)"/>
+  </svg><span class='ml-4'>${name}</span></div>`
+}
+
+function renderDescriptionRow(description: string) {
+  return `<div class="max-w-[216px] text-left">${description}</div>`
+}
+
+function formatDate(dateString: string): string {
+  const date = new Date(dateString)
+  const year = date.getFullYear()
+  const month = date.toLocaleDateString('en', {
+    month: 'short',
+  })
+  const day = date.toLocaleDateString('en', {
+    day: 'numeric',
+  })
+
+  const ending = getOrdinalSuffix(date.getDate())
+
+  return `${year} ${month} ${day}${ending}`
+}
+
+function getOrdinalSuffix(days: number) {
+  if (days > 3 && days < 21) return 'th'
+  switch (days % 10) {
+    case 1:
+      return 'st'
+    case 2:
+      return 'nd'
+    case 3:
+      return 'rd'
+    default:
+      return 'th'
+  }
+}
```

Comment:
We should have consistent date formatting. Either use this new format everywhere or use the old format here.